projects/1.20/assets/architects-palette/architects_palette/lang/zh_cn.json

```
@@ -0,0 +1,643 @@
+
+
+  "__comment__": "These strings are used in the JEI/REI plugins.",
+  "architects_palette.info.warping_recipe_title": "移形",
+  "__comment__": "EMI implementation detail. Should be the same string as above.",
+  "emi.category.architects_palette.warping": "移形",
+  "__comment__": "The %s in the string is replaced with the corresponding dimension when displayed in game",
+  "architects_palette.info.warping_toss_description": "将其扔进一个连通至%s的传送门",
+  "__comment__": "Modders can add support to the preview by adding something like this to their lang files, under the architects_palette namespace",
+  "architects_palette.info.dimension.minecraft.the_nether": "下界",
+  "architects_palette.info.dimension.minecraft.the_end": "末地",
+  "architects_palette.info.dimension.minecraft.overworld": "主世界",
+  "architects_palette.info.dimension.aether.the_aether": "天境",
+  "architects_palette.info.chiseled_chargeable": "在其中放入海洋之心以给它和所有相邻同类方块供能。",
+  "architects_palette.info.chargeable": "这些方块能从錾制渊晶石砖中汲取能量，更多信息请查询錾制渊晶石砖。",
+  "architects_palette.info.totem_carving": "在图腾上使用斧以切换图腾样式。",
+  "architects_palette.info.flint_damage": "燧石很硬！生物摔落到燧石方块上时受到的摔落伤害增加25%。",
+  "architects_palette.info.celestial_stones": "这些石头会根据当前时间改变自身形态和亮度。在所有维度中都有效。",
+  "architects_palette.info.heavy_bricks": "每两个相邻方块会连接在一起，便于制造大型砖块状纹理。",
+  "architects_palette.info.cage_lanterns": "可由红石信号点亮。控制模式可通过右击反转。",
+  "architects_palette.info.nether_brass": "下界黄铜被点燃时，其上火焰会呈现出独特的绿色。",
+  "architects_palette.info.wardstone": "青卫石方块会弹开射到其上的弹射物。",
+
+  "architects_palette.advancement.root": "建筑师的调色板",
+  "architects_palette.advancement.root.desc": "是时候建点新的东西了。",
+  "architects_palette.advancement.totem_carving": "让你笑不出来",
+  "architects_palette.advancement.totem_carving.desc": "用斧将一个金合欢木图腾雕刻成另一种样式。",
+  "architects_palette.advancement.buy_fish_block": "好大一条鱼！",
+  "architects_palette.advancement.buy_fish_block.desc": "与渔夫村民交易得到一块新鲜的鱼肉。",
+  "architects_palette.advancement.buy_pipe": "本地的管道工",
+  "architects_palette.advancement.buy_pipe.desc": "从工具匠手里买下一些管道方块。",
+  "architects_palette.advancement.buy_entrails": "精细地屠宰",
+  "architects_palette.advancement.buy_entrails.desc": "用你宝贵的绿宝石换一块……肠子？",
+  "architects_palette.advancement.place_entrails": "不知怎么地，比想象的更糟",
+  "architects_palette.advancement.place_entrails.desc": "放下并注视你新买回来的肠子。",
+  "architects_palette.advancement.buy_plating": "厚厚的钢板",
+  "architects_palette.advancement.buy_plating.desc": "与盔甲匠村民交易得到一些他们的金属板",
+  "architects_palette.advancement.buy_spool": "批量采购",
+  "architects_palette.advancement.buy_spool.desc": "买一卷牧羊人的羊毛，或者你自己合成一卷。",
+  "architects_palette.advancement.buy_celestial_stone": "想买个钟表吗？",
+  "architects_palette.advancement.buy_celestial_stone.desc": "从流浪商人手中购买一些日长石或者月长石。",
+  "architects_palette.advancement.find_warpstone": "这是什么东西？",
+  "architects_palette.advancement.find_warpstone.desc": "将一个黏土块扔过下界传送门，然后在传送门的另一侧发现不一样的东西。",
+  "architects_palette.advancement.find_twisted_sapling": "现实扭曲",

```

```suggestion
  "architects_palette.advancement.find_twisted_sapling": "歪歪扭扭",
```
原文并无“现实扭曲”如此严肃，反倒更多是在用日常闲杂用语，建议用诙谐的四字词语代替